packages/frontend/src/components/EpochKeyEvent.jsx

```diff
@@ -35,7 +35,7 @@ export default observer(({ attestation }) => {
         )}
       </div>
       <p style={{ minWidth: '100px', fontSize: '12px' }}>
-        {dayjs(timestamp * 1000).from(dayjs())}
+        {dayjs(+blockTimestamp * 1000).from(dayjs())}
       </p>
     </div>
```

Comment:
add arrow and link here as well?

Reply:
yep, thanks for all the above comments, updated!